passl/datasets/preprocess/transforms.py

```diff
@@ -20,7 +20,7 @@
 
 import paddle.vision.transforms as PT
 import paddle.vision.transforms.functional as F
-
+from .cv2_trans import ByolRandomHorizontalFlip, ByolColorJitter, ByolRandomGrayscale, ByolNormalize,ToCHW,ToRGB,ByolCenterCrop, ByolRandomCrop
```

Comment:
It's too long, how about this:
from .cv2_trans import ByolRandomHorizontalFlip, ByolColorJitter,\
    ByolRandomGrayscale, ByolNormalize,ToCHW,ToRGB,ByolCenterCrop, ByolRandomCrop
install pre_commit can check and fix this problem automaticly.